spec/system/form_submission_spec.rb

```diff
@@ -75,6 +77,15 @@
       click_on "Complete"
 
       expect(page).to have_content "awaiting_approval"
+
+      # Now sign is as the collection curator and see the notification on your dashboard
+      sign_out user
+      sign_in curator
+      visit(user_path(curator))
+      expect(page).to have_content curator.display_name
+      within("#unfinished_datasets span.badge.rounded-pill.bg-primary") do
+        expect(page).to have_content "1"
+      end
```

Comment:
```suggestion
      within("#unfinished_datasets") do
        expect(page).to have_content "Sowing the Seeds for More Usable Web Archives: A Usability Study of Archive-It"
        expect(page).to have_content "awaiting_approval"
      end
```
The exact element path doesn't seem like a requirement, but seeing the title and the status does.

Reply:
@mccalluc I'm looking for the badge with the 1 here.  I'll add a comment